mmedit/datasets/pipelines/crop.py

```diff
@@ -39,6 +45,24 @@ def _crop(self, data):
         for item in data_list:
             data_h, data_w = item.shape[:2]
             crop_h, crop_w = self.crop_size
+
+            crop_y_offset, crop_x_offset = 0, 0
+
+            if crop_h > data_h:
+                crop_y_offset = (crop_h - data_h) // 2
+            elif crop_w > data_w:
+                crop_x_offset = (crop_w - data_w) // 2
+
+            if crop_y_offset > 0 or crop_x_offset > 0:
+                pad_width = [(2 * crop_y_offset, 2 * crop_y_offset),
+                             (2 * crop_x_offset, 2 * crop_x_offset)]
+                if item.ndim == 3:
+                    pad_width.append((0, 0))
```

Comment:
If the padding is the same for two sides, we do not need to use a tuple, simply
```
pad_width = [2 * crop_y_offset, 2 * crop_x_offset]
if item.ndim == 3:
    pad_width.append(0)
```
will do the job.

Reply:
doing so will result in the following error: `ValueError: operands could not be broadcast together with remapped shapes [original->remapped]: (3,)  and requested shape (3,2)`.

Reply:
Oops you are correct, my bad haha.